pages/faq.md

```diff
@@ -41,10 +40,32 @@ Please make sure you’re using Firefox, as our BeLL software is only guaranteed
 *Q: Why does Firefox say “Unable to connect” when I try to load my community?*
 Because a community is run locally on your machine, you need to `vagrant up` in the directory where the Vagrantfile is located. You can then see if your community is running by going to `127.0.0.1:5985` in Firefox. Go to `127.0.0.1:5985/_utils` to see the CouchDB behind the BeLL, and `127.0.0.1:5985/apps/_design/bell/MyApp/index.html` to navigate the actual BeLL user interface.
 
+*Q: When I first run BeLL with the `vagrant up` command the download fails, why?*
+You should first check if [Hashicorp's atlas](https://atlas.hashicorp.com/boxes/search) is up and running by looking at the left sidebar :
+
+![left sidebar](uploads/images/atlas_status.png)
+
+If Atlas is operational, then maybe your download is being interrupted, you can try to download and set up the big vagrant box file manually:
+
+1. Go to this [atlas box page](https://atlas.hashicorp.com/ole/boxes/jessie64/)
+
+2. Click on the last version's (the uppermost) version number
```

Comment:
Please fix that
![screen shot 2017-02-15 at 1 13 30 pm](https://cloud.githubusercontent.com/assets/16807118/22993640/eaa702f0-f380-11e6-822e-adbd819edfbd.png)